MusicNotationKit/MusicNotationKit/Clef.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  Clef.swift
+//  MusicNotationKit
+//
+//  Created by Kyle Sherman on 10/16/16.
+//  Copyright © 2016 Kyle Sherman. All rights reserved.
+//
+
+public struct Clef {
+
+    public let tone: Tone?
+    /**
+     Starts from 0 on the first line (from the bottom). Ledger lines below that are negative.
+     Each increase by 1 moves a half step. i.e. 1 is the first space on the staff.
+     */
+    public let lineNumber: Int
```

Comment:
Yeah, I think that would probably be the most self-explanatory:

`public convenience init(tone: Tone, line: Int)`
`public convenience init(tone: Tone, space: Int)`

So the lines of the staff would be 0-4, and the spaces 0-3.